### DISCUSSION TITLE
Why is WinSCP listed as a pre-req in the README?

### DISCUSSION THREAD

**Discussion by bharloe** [Nimdy/Dedicated_Valheim_Server_Script#165] [Q&A] [answered]
Title pretty much says it all, I am just trying to figure out what I need WinSCP for here. I will be running an Ubuntu server so my guess is this doesn't apply to that but I can't be sure. I also don't see any other reference to WinSCP in this repo other than in [that section of the README](https://github.com/Nimdy/Dedicated_Valheim_Server_Script/blob/main/README.md#prerequisites).

**Comment by Nimdy** [comment]
best 95% of the time... the next question I get asked... How do you transfer you worlds?

**Reply by rautamiekka** [reply]
What could WinSCP, a Window$-only app, do FileZilla can't ?

**Reply by devdavi** [reply]
For this purpose, probably nothing, but most users are on Windows. Since you're commenting on OS specific programs, you probably already know that. I personally wouldn't recommend FileZilla to anyone as it (or it's developer) has had a rocky history of poor security practices. Which is probably why the recommendation wasn't made in the first place. 

If you want a true cross-platform way to connect to the server and retrieve files I'd recommend the terminal, or Windows Terminal if you happen to find yourself on Window$

**Comment by devdavi** [comment, marked as answer]
WinSCP is just recommended as an easier to use method to access files directly/via GUI. 

@Nimdy has updated the Readme to clarify the purpose of WinSCP, thanks for pointing out the confusion with that part.

**Reply by rautamiekka** [reply]
Easier, if you're on Window$ ... -.-

**Reply by Nimdy** [reply]
Thanks Devdavi!
